apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -1086,7 +1086,7 @@ export class InstallManager {
           if (environmentVariables[envVar].override &&
             environmentVariables[envVar].override === true) {
             setEnvironmentVariable = true;
-            console.log(colors.yellow(`WARNING: Overriding the environment variable with the value set in rush.json.`));
+            console.log(`Overriding the environment variable with the value set in rush.json.`);
```

Comment:
Maybe this should only be printed in `--debug` mode.

Reply:
I think this should always be printed. I don't want rush to silently override a variable set on the command line only for the session or in a ci job step.

Reply:
Hmm okay. Let's see how chatty it ends up being.